src/analysis/ContextBar.test.ts

```diff
@@ -379,12 +377,7 @@ const runtimeDisk: DecoratedPersistentDisk = {
   },
 };
 
-const defaultGoogleBucketOptions = {
-  googleBucketLocation: defaultLocation,
-  googleBucketType: locationTypes.default,
-  fetchedGoogleBucketLocation: undefined,
-};
-const defaultAzureStorageOptions = {
+const populatedAzureStorageOptions = {
```

Comment:
I initially extracted `defaultAzureStorageOptions` as well up into the shared workspace-fixtures module I added for this, but after some feedback from @cahrens concluded that calling these defaults in that shared domain isn't accurate.  The defaults are actually all empty.

So I updated `defaultAzureStorageOptions` to clarify that the fields are unset, and renamed this to disambiguate & distinguish it from the defaults.